recipes/perl-gd/meta.yaml

```diff
@@ -14,15 +14,13 @@ requirements:
     - llvm # [osx]
     - perl-threaded >=5.22.0
     - libgd >=2.2.3
-    - fontconfig # for libfontcontig.so
```

Comment:
I dont think so, but we can try if it fails again. Last fail was most likely due to an ancient dep build...